Source/Atomic/UI/UI.h

```diff
@@ -100,6 +100,9 @@ class UI : public Object, private tb::TBWidgetListener
     void ShowDebugHud(bool value);
     void ToggleDebugHud();
 
+    // Cycle debug HUD between showing primitive stats, current mode, profiler data, all three or none
+    void CycleDebugHudMode();
+
```

Comment:
There's actually a mishmash of commenting standards in that header alone. //, ///, and comments starting capitalized and non-capitalized